frontend/src/lib/taxonomy.tsx

```diff
@@ -1011,17 +1078,12 @@ for (const [key, value] of Object.entries(CORE_FILTER_DEFINITIONS_BY_GROUP.event
     } else {
         CORE_FILTER_DEFINITIONS_BY_GROUP.person_properties[key] = value
     }
-    if (SESSION_PROPERTIES_ADAPTED_FROM_EVENT.has(key)) {
-        CORE_FILTER_DEFINITIONS_BY_GROUP.sessions[`$initial_${key.replace(/^\$/, '')}`] = {
-            ...value,
-            label: `Initial ${value.label}`,
-            description:
-                'description' in value
-                    ? `${value.description} First value seen in the session.`
-                    : 'First value seen in the session.',
-        }
-    }
 }
+
+for (const key of Array.from(SESSION_PROPERTIES_ADAPTED_FROM_PERSON.keys())) {
+    CORE_FILTER_DEFINITIONS_BY_GROUP.sessions[key] = CORE_FILTER_DEFINITIONS_BY_GROUP.person_properties[key]
```

Comment:
Does "Data from the first time this user was seen." also apply to these session-level $initial_ properties?

Reply:
It is fixed